server/src/main/scala/za/co/absa/atum/server/model/GetCheckpointV2Args.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.model
+
+import java.util.UUID
+
+case class GetCheckpointV2Args(
```

Comment:
I'm not a fan of the placement of this one. What about moving it to `GetPartitioningCheckpointV2.scala` ? Similarly as here: https://github.com/absa-group/AUL/pull/2844/files#diff-e4b451049f88dfb52b116688e6b5e7e7baec175ce718a821d513d1b02aa75da2, just put it into the companion object of the function. I really like it - it's much more closer together and thus we can avoid having these random small case classes in service's internal model

Reply:
Again, let's discuss with others. In general I am not against placing these in those companions but I can also imagine cases where we use the same structure for several database functions and then it wouldn't be appropriate to import from one fa-db object to another one. Personally I would rather create a separate package for the args classes.